### DISCUSSION TITLE
Moose Mesh generator code error

### DISCUSSION THREAD

**Discussion by libinbabu1991** [idaholab/moose#28893] [Q&A Meshing]
Hello, 
I am following along the MOOSE workshop tutorials. (https://www.youtube.com/watch?v=QPuK6OdF2hM)
I couldn't generate the mesh as shown by the instructor and getting below error. 
 
 (moose) root@131-ENG-DESKTOP:~/projects/moose/tutorials/darcy_thermo_mech/step01_diffusion/problems# ../darcy_thermo_mech-opt -i step1_logan.i --mesh-only

I appreciate any guidance on this.

*** ERROR ***
/root/projects/moose/tutorials/darcy_thermo_mech/step01_diffusion/problems/step1_logan.i:1.1: section '[mesh]' does not have an associated "Action".
 Common causes:
- you misspelled the Action/section name
- the app you are running does not support this Action/syntax
/root/projects/moose/tutorials/darcy_thermo_mech/step01_diffusion/problems/step1_logan.i:2.3: section '[mesh/mesh]' does not have an associated "Action".
 Common causes:
- you misspelled the Action/section name
- the app you are running does not support this Action/syntax

**Comment by GiudGiud** [comment]
Hello

[mesh] should like be [Mesh]

Guillaume

**Reply by libinbabu1991** [reply]
Just read the Syntax rules. It worked, thank you very much!
